src/data/navigation.ts

```diff
@@ -112,6 +112,21 @@ export const navigation: NavigationItem[] = [
           {
             title: "Search Examples", href: "/code-search/examples",
           },
+          {
+            title: "Code Navigation", href: "/code-search/code-navigation",
+            subsections: [
+              { title: "Capabilities", href: "/code-search/code-navigation/features", },
+              { title: "Search Navigation", href: "/code-search/code-navigation/search_based_code_navigation", },
+              { title: "Precise Navigation", href: "/code-search/code-navigation/precise_code_navigation", },
```

Comment:
```suggestion
              { title: "Search-based code navigation", href: "/code-search/code-navigation/search_based_code_navigation", },
              { title: "Precise code navigation", href: "/code-search/code-navigation/precise_code_navigation", },
```

Nit: IIUC, these will show up in a sidebar or similar. If so, please rename these to avoid terms "Search navigation" or "Precise navigation" which are not used elsewhere else in the documentation.